src/main/java/org/openstreetmap/atlas/exception/change/FeatureChangeMergeException.java

```diff
@@ -0,0 +1,34 @@
+package org.openstreetmap.atlas.exception.change;
+
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.geography.atlas.change.FeatureChange;
+
+/**
+ * A special exception for {@link FeatureChange} merge errors.
+ * 
+ * @author lcram
+ */
+public class FeatureChangeMergeException extends CoreException
+{
+    private static final long serialVersionUID = -3583945839922744755L;
+
+    private final transient MergeFailureType failureType;
```

Comment:
It would be nice to not make it transient so it can be serialized around...

Reply:
Done.